ceilometer/manifests/init.pp

```diff
@@ -181,9 +181,9 @@
     }
 
       if size($rabbit_hosts) > 1 {
-        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => true }
+        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => undef }
       } else {
-        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => false }
+        ceilometer_config { 'oslo_messaging_rabbit/rabbit_ha_queues': value => undef }
       }
```

Comment:
there is only ever one "rabbit_hosts", you're right. I switched this back to be false and will include it in next commit